.github/workflows/build.yml

```diff
@@ -68,13 +70,18 @@ jobs:
       - name: Test
         run: |
           doit test
+      - name: Docs
+        run: |
+          doit docs
+      - name: Check Built Artifacts
+        run: |
+          doit check
 
   deploy:
+    if: github.ref == 'refs/heads/main'
     needs: [build]
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
```

Comment:
iirc, this step is needed for the action below, which was failing otherwise (while doing #68) because of:

https://github.com/amondnet/vercel-action/blob/006f00badc059602e3d10a9862923d54552c621b/index.js#L339